### PR TITLE
fix: set `TARGET_CC` env var on Windows using unix spearator

### PR DESCRIPTION
**Pull request by amrbashir**
this helps cross-compliation

### CODE REVIEW

File: src/android/target.rs [hunk 1]
```diff
@@ -240,6 +240,34 @@ impl<'a> Target<'a> {
...
+            .ndk
+            .compiler_path(ndk::Compiler::Clang, self.clang_triple(), min_sdk_version)
+            .map_err(CompileLibError::MissingTool)?
+            .display()
+            .to_string()
+            .replace("\\", "/");
+        #[cfg(windows)]
+        let clangxx_path = env
+            .ndk
+            .compiler_path(ndk::Compiler::Clang, self.clang_triple(), min_sdk_version)
```
Comment (lucasfernog) [inline src/android/target.rs]:
```suggestion
            .compiler_path(ndk::Compiler::Clangxx, self.clang_triple(), min_sdk_version)
```

File: src/android/target.rs [hunk 2]
```diff
@@ -240,6 +240,34 @@ impl<'a> Target<'a> {
...
 
+        #[cfg(unix)]
+        let clang_path = env
+            .ndk
+            .compiler_path(ndk::Compiler::Clang, self.clang_triple(), min_sdk_version)
+            .map_err(CompileLibError::MissingTool)?;
+        #[cfg(unix)]
+        let clangxx_path = env
+            .ndk
+            .compiler_path(ndk::Compiler::Clang, self.clang_triple(), min_sdk_version)
```
Comment (lucasfernog) [inline src/android/target.rs]:
```suggestion
            .compiler_path(ndk::Compiler::Clangxx, self.clang_triple(), min_sdk_version)
```

**Review by lucasfernog — CHANGES_REQUESTED** [review]

**Comment by Ludea** [comment]
I just test your PR, the latest `/` is missing
https://github.com/Ludea/tauri-mobile/actions/runs/3054766151/jobs/4927396987#step:16:366

**Comment by amrbashir** [comment]
yeah it seems like an issue upstream and we can't fix it here. I will probably close this PR since it fixed nothing

**Comment by Ludea** [comment]
maybe changing https://github.com/tauri-apps/cargo-mobile/blob/dev/src/os/linux/mod.rs#L191 to `pub const CLANG: &str = "/clang";` can help ?

**Comment by amrbashir** [comment]
I don't think so plus it could break cross-compilation in other places

**Pull request closed without merging**